infrastructure/template.yaml

```diff
@@ -1224,6 +1254,47 @@ Resources:
         - Name: StateMachineArn
           Value: !Ref BearerTokenRetrievalStateMachine
 
+  BearerTokenRetrievalStateMachineExecutionFailedCanaryAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !ImportValue platform-alarm-warning-alert-topic
+      OKActions:
+        - !ImportValue platform-alarm-warning-alert-topic
+      AlarmName: !Sub ${AWS::StackName}-${Environment}-BearerTokenRetrieval-ExecutionFailedCanaryAlarm
+      AlarmDescription: !Sub "ExecutionFailed error returned from the BearerTokenRetrievalStateMachine"
+      ComparisonOperator: GreaterThanThreshold
+      EvaluationPeriods: 1
+      MetricName: ExecutionsFailed
+      Namespace: AWS/States
+      Period: 60
+      Statistic: Sum
+      Threshold: 0
```

Comment:
should be 1?

Reply:
`ComparisonOperator: GreaterThanThreshold` so 0 is technically fine here - however, I've updated just for consistency! (there's a mix of both in the template)